From b5971a4c9aaff12c408893eb9294b41cf1c7435d Mon Sep 17 00:00:00 2001
From: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Date: Mon, 13 Jan 2014 10:31:49 +0000
Subject: [PATCH 083/351] ASoC: wm_adsp: Route virtual inputs through
 preloader

All inputs to the DSP need to go through the preloader and this change
was missed for the virtual inputs on cores 2 and 3 when the firmware
loading was split out.

Change-Id: Ie7504c8b375a6b347bbb960a81348ceda25ecb23
Signed-off-by: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Signed-off-by: Nikesh Oswal <Nikesh.Oswal@wolfsonmicro.com>
---
 sound/soc/codecs/florida.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/codecs/florida.c b/sound/soc/codecs/florida.c
index 0df32b7..9fa560d 100644
--- a/sound/soc/codecs/florida.c
+++ b/sound/soc/codecs/florida.c
@@ -1438,9 +1438,9 @@ static const struct snd_soc_dapm_route florida_dapm_routes[] = {
 	ARIZONA_DSP_ROUTES("DSP3"),
 	ARIZONA_DSP_ROUTES("DSP4"),
 
-	{ "DSP2",  NULL, "DSP2 Virtual Input" },
+	{ "DSP2 Preloader",  NULL, "DSP2 Virtual Input" },
 	{ "DSP2 Virtual Input", "Shared Memory", "DSP3" },
-	{ "DSP3", NULL, "DSP3 Virtual Input" },
+	{ "DSP3 Preloader", NULL, "DSP3 Virtual Input" },
 	{ "DSP3 Virtual Input", "Shared Memory", "DSP2" },
 
 	{ "DSP Virtual Output", NULL, "DSP Virtual Output Mux" },
-- 
1.7.9.5

